guides/common/modules/con_managing-alternate-content-sources.adoc

```diff
@@ -0,0 +1,10 @@
+[id="Managing_Alternate_Content_Sources_{context}"]
+= Managing Alternate Content Sources
+
+Alternate content sources define alternate paths to download content during synchronization.
+The content itself is downloaded from the alternate content source and only the metadata is downloaded from the upstream URL.
+You can use alternate content source to speed up synchronization if the content is located on the local filesystem or on a nearby network.
+You can set up alternate content sources for {ProjectServer} and {SmartProxy}.
+You must refresh the alternate content source after creation or after making any changes.
```

Comment:
@maximiliankolb @ianballou does this look good?

Reply:
Works for me!

Reply:
Nice!